ludwig/automl/automl.py

```diff
@@ -48,16 +56,36 @@ def auto_train(dataset: str, target: str, time_limit_s: Union[int, float], outpu
     model_config, model_name = model_select(default_configs)
     hyperopt_results = _train(model_config, dataset,
                               output_dir, model_name=model_name)
+    experiment_analysis = hyperopt_results.experiment_analysis
+
+    # catch edge case where metric_score is nan
+    # TODO (ASN): Decide how we want to proceed if at least one trial has
+    # completed
+    for trial in hyperopt_results.ordered_trials:
+        if np.isnan(trial.metric_score):
+            raise ValueError(
+                "There was an error running the experiment. "
+                "A trial failed to start. "
+                "Consider increasing the time budget for experiment. "
```

Comment:
Are we sure failing to start is the only possible reason for a NaN?

Reply:
@w4nderlust  Not sure - let me investigate

Reply:
Another way around it is to just check if the `training_stats` and `eval_stats` are empty dicts.